config/env/production.js

```diff
@@ -78,6 +78,13 @@ module.exports = {
     callbackURL: '/api/auth/paypal/callback',
     sandbox: false
   },
+  owasp: {
+      allowPassphrases       : true,
+      maxLength              : 128,
+      minLength              : 10,
+      minPhraseLength        : 20,
+      minOptionalTestsToPass : 4,
+  },
```

Comment:
I'm fine moving this to the default config. The problem is it may break some tests until they can be updated to pull from the config. Right now some of the tests like password length are hard coded in the tests. I forgot to update the test config, but I'll be updating the PR shortly.